client-ts/typings/EventSource.d.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+// Hand-written EventSource typings. I couldn't find anything easy-to-consume out there. This is purely based on the API docs.
+// -anurse
+
+interface EventSourceOptions {
+    withcredentials: boolean
+}
+
+declare class EventSource extends EventTarget {
+    constructor(url: string);
+    constructor(url: string, configuration: EventSourceOptions);
+
+    readonly CLOSED: number;
```

Comment:
Move constants to an enum? https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
This is a declaration file for an existing browser API, not our own code :). It's actually not needed in `dev` because we updated TypeScript to a version that includes this declaration (it's the client API for Server-Sent Events in the browser)